iac/example.yml

```diff
@@ -0,0 +1,33 @@
+# A sample workflow which checks out your Infrastructure-as-Code Configuration files,
+# such as Kubernetes, Helm & Terraform and scans them for any security issues.
+# The results are then uploaded to GitHub Security Code Scanning
+#
+# For more examples, including how to limit scans to only high-severity
+# issues, monitor images for newly disclosed vulnerabilities in Snyk and
```

Comment:
`images` here should be something else. And monitor isn't applicable to `snyk iac`